### DISCUSSION TITLE
How do you navigate, edit, and reference your cards quickly?

### DISCUSSION THREAD

**Discussion by Daoet** [st3v3nmw/obsidian-spaced-repetition#863] [Q&A]
I make flashcards tagging them by topic.

However there are some things I couldn't find a more streamlined process for:
- navigation (undo): If I accidentally press the spacebar an extra time it will continue to the next card, leaving me without an option to either revert the scheduling due date or to undo my answer.
(In Anki I would just press a button to go back to the previous flashcard)

- reference and back (preview): when I need more context, I press "Open Note", it opens a tab which I access after closing the review window, then go to a note for the context. Then to continue reviewing that same card (mainly for the purpose of previewing the card I've edited) I need to open the review window again but unfortunately it displays randomized cards (i.e. not my last card review). It is a tedious process for me to go through the review session just to look for that card and definitely breaks the flow of reviewing.

- search & editing specific cards: I lack the feature to have an overview of all/selective cards related to the topic/tag and decide which one I want to edit. Sometimes I forget if I haven't already turned some specific prompt into a flashcard and I need a way to make sure it's included in my reviews.
(by the way, cram is not the feature I'm looking for because it only selects your deck, you have to go through your flashcards individually otherwise to get this specific flashcard, that won't do when you have tons of flashcards in one (sub)deck. I need something faster and more convenient. (this would equal the Browser in Anki where you could find your cards)

Are there ways to achieve what I want when reviewing my flashcards the way I'm doing, or are there workarounds? 
How do you review your flashcards w/o worrying about this process?

**Comment by ronzulu** [comment]
Hi @Daoet 

> navigation (undo): If I accidentally press the spacebar an extra time it will continue to the next card, leaving me without an option to either revert the scheduling due date or to undo my answer.

Best if you could add a note to #250 to say that you too would find it useful

> reference and back (preview): when I need more context, I press "Open Note", it opens a tab which I access after closing the review window, then go to a note for the context. Then to continue reviewing that same card (mainly for the purpose of previewing the card I've edited) I need to open the review window again but unfortunately it displays randomized cards (i.e. not my last card review). It is a tedious process for me to go through the review session just to look for that card and definitely breaks the flow of reviewing.

In terms of context, just checking that you know about the context option:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/97ac667b-51cb-4fc9-8e18-33efae0ddcb7)

(Also there have been requests for making the context info more flexible, #522)

I don't think that #786  is exactly what you are after, but thought you might be interested.

I think it would be straightforward to modify current functionality so that if a user doesn't complete a review, when they restart it goes back to the same card. As you point out, this would need to work even if any of the review ordering options were set to random. If that would be helpful, feel free to create an issue at:
https://github.com/st3v3nmw/obsidian-spaced-repetition/issues

> search & editing specific cards: I lack the feature to have an overview of all/selective cards related to the topic/tag and decide which one I want to edit. Sometimes I forget if I haven't already turned some specific prompt into a flashcard and I need a way to make sure it's included in my reviews.

Have a look at #857. If this doesn't cover what you have in mind, feel free to create a new feature request.

Cheers
Ronny
